common/saber_base.h

```diff
@@ -77,6 +77,7 @@ extern SaberBase* saberbases;
     DEFINE_EFFECT(UNJAM)			\
     DEFINE_EFFECT(PLI_ON)			\
     DEFINE_EFFECT(PLI_OFF)                      \
+    DEFINE_EFFECT(DESTRUCT)                     \
```

Comment:
self-destruct is usually an off-type rather than an effect...

Reply:
It is,
```
        case NEXT_ACTION_BLOW:
          Off(OFF_BLAST);
          break;
  ```
  But having EFFECT_DESTRUCT allows for blade animations.